src/modules/cart/repositories/cartRepositories.ts

```diff
@@ -1,24 +1,34 @@
-import db from "../../../databases/models"
+/* eslint-disable comma-dangle */
+import db from "../../../databases/models";
 
 const getCartsByUserId = async (userId: string) => {
-  return await db.Carts.findAll({ where: { userId, status: "pending" } })
-}
+  return await db.Carts.findAll({ where: { userId, status: "pending" } });
+};
+
+const getCartByUserId = async (userId: string) => {
```

Comment:
This function is useless because it picks random buyer's cart because buyer have many carts.